bounties/other/frokcms/1/vulnerability.json

```diff
@@ -0,0 +1,54 @@
+{
+    "PackageVulnerabilityID": "1",
+    "DisclosureDate": "2020-09-13",
+    "AffectedVersionRange": "*",
+    "Summary": "XSS",
+    "Contributor": {
+        "Discloser": "b1nslashsh",
+        "Fixer": ""
+    },
+    "Package": {
+        "Registry": "other",
+        "Name": "Forkcms",
+        "URL": "https://github.com/forkcms/forkcms",
+        "Downloads": "0"
+    },
+    "CWEs": [
+        {
+            "ID": "494",
+            "Description": ""
+        }
+    ],
+    "CVSS": {
+        "Version": "3.1",
+        "AV": "N",
+        "AC": "H",
+        "PR": "N",
+        "UI": "R",
+        "S": "C",
+        "C": "L",
+        "I": "L",
+        "A": "N",
+        "E": "",
+        "RL": "",
+        "RC": "",
+        "Score": "4.7"
```

Comment:
`Stored XSS` on `auth page` should be considered == `account takeover`.
I would put `8.1`